### ISSUE TITLE
Document various version tags

### ISSUE THREAD

**Issue by ThomasLaPiana** [ethyca/fides#3553]
### Docs Update Description

We have multiple version tags, including `alpha`, `beta`, `rc`, etc.

These should be fully documented in our developer docs so we know when to use what and why

**Comment by ThomasLaPiana** [comment]
@adamsachs any chance we could tackle this together? Although I have a feeling you've already done this somewhere...

**Comment by adamsachs** [comment]
yes @ThomasLaPiana ! we've got a couple of different docs floating around, but let's work together to see if we can improve/consolidate/restructure any of it.

here's what we've got so far:
- https://github.com/ethyca/fidesplus/blob/main/README.md#core-oss-fides-integration
- https://ethyca.atlassian.net/wiki/spaces/EN/pages/2686189724/Using+nox+-s+tag+to+help+with+integrated+fides+fidesplus+development
